src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -389,7 +390,7 @@ public CookieAcceptPolicy CookieAcceptPolicy
         /// <summary>
         /// The user agent string.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
Please keep [EditorBrowsable(EditorBrowsableState.Never)] in DevelNUI branch.
When this patch is merged, you can remove it on Master branch with ACR.